apps/rand.c

```diff
@@ -25,7 +25,7 @@ typedef enum OPTION_choice {
 } OPTION_CHOICE;
 
 const OPTIONS rand_options[] = {
-    {OPT_HELP_STR, 1, '-', "Usage: %s [options] num\n"},
+    {OPT_HELP_STR, 1, '-', "Usage: %s [options] num[k|m|g|t]\n"},
```

Comment:
I would make the m/g/t here capitals.

Reply:
I completely agree with @hlandau: everybody is used to 1k 2M 3G ... 

And whoever doesn't like it can be pleased as well, because the suffix is case-insensitive.

Reply:
fixed capitalization.